src/Microsoft.AspNetCore.Server.Kestrel/Http/Frame.cs

```diff
@@ -696,8 +696,129 @@ private Task WriteSuffix()
             }
         }
 
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
How important is `AggressiveInlining`?  We currently don't use it anywhere in Kestrel.